client/App/Sagas/SamplesSaga.js

```diff
@@ -1,16 +1,20 @@
 import { race, take, select, put, all, delay, fork, cancel, call } from 'redux-saga/effects';
 import GpsActions, { GpsTypes } from '../Stores/Gps/Actions';
-import { NEXT_SAMPLE_DELAY as DELAY } from '../Consts';
+import { NEXT_SAMPLE_DELAY , ANDROID10_SAMPLE_DELAY, ANDROID10_API_LEVEL as ANDROID10} from '../Consts';
 import WifiActions, { WifiTypes } from '../Stores/Wifi/Actions';
 import SampleActions from '../Stores/Samples/Actions';
 import { gpsService } from '../Services/GpsService';
 import { wifiService } from '../Services/WifiService';
 import APIService from '../Services/APIService';
+import { Platform } from 'react-native';
 
 const wifiListSelector = (state) => !state.wifi.sampleSent && state.wifi.wifiList;
 const gpsLocationSelector = (state) => !state.gps.sampleSent && state.gps.gpsLocation;
+const DELAY = 
+Platform.Version >= ANDROID10 ? ANDROID10_SAMPLE_DELAY : NEXT_SAMPLE_DELAY;
```

Comment:
Great!!! Loved the play. You made the change as little as possible by importing Next_sample_delay as the original name. Perfect!